devicetypes/axis/axis-gear-v5.1.src/axis-gear-v5.1.groovy

```diff
@@ -0,0 +1,420 @@
+metadata {
+    definition (name: "AXIS Gear-V5.1", namespace: "axis", author: "AXIS Labs") {  
+        capability "Window Shade"
+        capability "Window Shade Preset"
+        capability "Switch Level"
+        capability "Battery"
+		capability "Refresh"
```

Comment:
Repeating the general request to make whitespace consistent. Here's an example of a line using tabs immediately following a line using spaces.